qa/tasks/cephfs/test_strays.py

```diff
@@ -799,110 +799,68 @@ def test_fancy_layout(self):
     def test_dirfrag_limit(self):
```

Comment:
and /ceph/teuthology-archive/pdonnell-2021-09-17_20:49:46-fs-wip-pdonnell-testing-20210917.174826-distro-basic-smithi/6395378/teuthology.log

Reply:
fixed